docs/DeveloperGuide.md

```diff
@@ -285,28 +285,27 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC02 - Find Client
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests enters search term
+2.  HustleBook shows a client whose name matches the given search term
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 2a. Search term does not match any client's name.
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 2b. Search term matches multiple client's name
 
-    * 3a1. AddressBook shows an error message.
+    * 3b1. AddressBook shows list of clients whose name matches search term
```

Comment:
Perhaps this should be 2b1.  since it is the first step of step 2b. instead of 3b.s

Reply:
Maybe it's good to also consider an extension 2c when user searches but the list is empty?